### PR TITLE
Update Process.WorkingSet64 docs and mention Refresh

### PR DESCRIPTION
**Pull request by adamsitnik**
https://github.com/dotnet/runtime/issues/46050#issuecomment-745104997

cc @svick

### CODE REVIEW

**Review by eiriktsarpalis — APPROVED** [review]

**Comment by danmosemsft** [comment]
Same for any other properties on Process?

File: xml/System.Diagnostics/Process.xml [hunk 1]
```diff
@@ -2989,7 +2989,9 @@ The calling process is a member of the associated process' descendant tree.</exc
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The value returned by this property represents the current size of nonpaged system memory used by the process, in bytes. System memory is the physical memory used by the operating system, and is divided into paged and nonpaged pools. Nonpaged memory allocations remain in system memory and are not paged out to the virtual memory paging file.  
+ The value returned by this property represents the most recently refreshed size of nonpaged system memory used by the process, in bytes. To get the current size, you need to call `process.Refresh()` method first.
```
Comment (svick) [inline xml/System.Diagnostics/Process.xml:2992]: I think this should link to the `Refresh` method:

```suggestion
 The value returned by this property represents the most recently refreshed size of nonpaged system memory used by the process, in bytes. To get the current size, you need to call <xref:System.Diagnostics.Process.Refresh> method first.
```

**Comment by adamsitnik** [comment]
> Same for any other properties on Process?

@danmosemsft good point, done

File: xml/System.Diagnostics/Process.xml [hunk 2]
```diff
@@ -2989,7 +2989,9 @@ The calling process is a member of the associated process' descendant tree.</exc
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The value returned by this property represents the current size of nonpaged system memory used by the process, in bytes. System memory is the physical memory used by the operating system, and is divided into paged and nonpaged pools. Nonpaged memory allocations remain in system memory and are not paged out to the virtual memory paging file.  
+ The value returned by this property represents the most recently refreshed size of nonpaged system memory used by the process, in bytes. To get the current size, you need to call <xref:System.Diagnostics.Process.Refresh> method first.
```
Comment (danmosemsft) [inline xml/System.Diagnostics/Process.xml:2992]: "To get the most up to date size, .." might be better wording. "To get the current size.." reads like, unless I call Refresh(), I might get bogus results or no results at all. (Sort of an oerload of meaning of "current")

**Review by danmosemsft — APPROVED** [review]
Note that the following properties on Process could have the same commen: 
ProcessorAffinity , PriorityBoostEnabled, PriorityClass, BasePriority, Threads, HandleCount

File: xml/System.Diagnostics/Process.xml (hunk 2 above)
Comment (eiriktsarpalis) [reply xml/System.Diagnostics/Process.xml:2992]: How about "To get a more recent size"?

**Comment by adamsitnik** [comment]
> Note that the following properties on Process could have the same commen:
ProcessorAffinity , PriorityBoostEnabled, PriorityClass, BasePriority, Threads, HandleCount

I've added the missing comments to the properties that you have listed and others that were missing it (based on current implementation)

**Review by danmosemsft — APPROVED** [review]

**Pull request merged**